Demo/Demo/DemoHelpers.swift

```diff
@@ -218,3 +218,21 @@ extension Array {
         return indices.contains(index) ? self[index] : .none
     }
 }
+
+extension Foundation.Notification.Name {
+    static let DidChangeUserInterfaceStyle = Foundation.Notification.Name("DidChangeUserInterfaceStyle")
```

Comment:
I think we usually don't use capitalized names for constants. Also, most of the notification names from Apple are not capitalized, e.g. https://developer.apple.com/documentation/appkit/nsaccessibility/notification/1527225-applicationactivated

Reply:
I was thinking about that, fixed on upcoming PR